yoga/Yoga.h

```diff
@@ -143,6 +143,7 @@ WIN_EXPORT void YGNodeCopyStyle(const YGNodeRef dstNode, const YGNodeRef srcNode
   WIN_EXPORT type YGNodeLayoutGet##name(const YGNodeRef node, const YGEdge edge);
 
 YG_NODE_PROPERTY(void *, Context, context);
+YG_NODE_PROPERTY(void *, Managed, managed);
```

Comment:
@emilsjolander technically we can use YGNodeSetContext instead of this. what do you think? this will keep a weak reference of YogaNode managed object.